karabo_data/geometry2.py

```diff
@@ -99,8 +99,11 @@ class AGIPD_1MGeometry:
     of the pixel size.
     """
     pixel_size = 2e-7  # 2e-7 metres == 0.2 mm
```

Comment:
pixel_size = 2e-4

Reply:
Yup, but you're already on that in PR #97 :+1: